lib/precompiled/08-ecpairing.js

```diff
@@ -4,17 +4,15 @@ const error = require('../exceptions.js').ERROR
 const fees = require('ethereum-common')
 const assert = require('assert')
 
-const bn128Module = require('rustbn.js')
-const ecPairingPrecompile = bn128Module.cwrap('ec_pairing', 'string', ['string'])
+const bn128 = require('rustbn.js')
 
 module.exports = function (opts) {
   assert(opts.data)
 
   let results = {}
   let data = opts.data
 
-  let inputHexStr = data.toString('hex')
-  let inputData = Buffer.from(inputHexStr, 'hex')
```

Comment:
This was some funny code :)

Reply:
https://en.m.wikipedia.org/wiki/Hungarian_notation

Reply:
I meant the fact in is doing `Buffer.from(data.toString('hex'), 'hex')`